harness/dotnet/Temporalio.Features.Harness/App.cs

```diff
@@ -79,19 +84,28 @@ private static async Task RunCommandAsync(InvocationContext ctx)
         var logger = loggerFactory.CreateLogger(typeof(App));
 
         // Connect a client
+        var tlsServerName = ctx.ParseResult.GetValueForOption(tlsServerNameOption);
+        TlsOptions? tlsOptions = null;
+        if (ctx.ParseResult.GetValueForOption(clientCertPathOption) is { } certPath)
+        {
+            tlsOptions = new()
+            {
+                ClientCert = File.ReadAllBytes(certPath.FullName),
+                ClientPrivateKey = File.ReadAllBytes(
+                    ctx.ParseResult.GetValueForOption(clientKeyPathOption)?.FullName ??
+                    throw new ArgumentException("Missing key with cert"))
+            };
+            if (!string.IsNullOrEmpty(tlsServerName))
```

Comment:
Probably don't need this conditional, can just assign to the field (I wouldn't expect it'd come across as empty)